generator/04-syndesis-rest.yml.mustache

```diff
@@ -26,7 +26,24 @@
       targetPort: 8080
     selector:
       app: syndesis
+      component: syndesis-rest{{#Probeless}}
```

Comment:
Actually, adding the route has nothing to do with _probeless_. Instead its a temporary route for direct access without authentication. 

We should rename `Probeless` to `Testing` to make clear that it is only for test purposes. Also, I don't think that disabling probes should stay forever. Hopefully, we can upgrade our cluster soon.


Please note that certain operation relies on that the authorization subject is set, which only happens when going through the oauth proxy. So your tests might then not be too realistic and might even fail.

Reply:
@rhuss could we merge this one and address the renaming in subsequent PR? For sure we need to investigate new oauth-proxy behaviour a bit further wrt to testing.

Reply:
ok, but we should not forget ;-)